docs/running-multiple-instances.md

```diff
@@ -0,0 +1,32 @@
+By default, Watchtower will clean up other instances and won't allow multiple instances monitoring the same containers. It is possible to override this behavior by defining a [scopeUID](https://containrrr.github.io/watchtower/arguments/#filter_by_scope) to each running instance. 
+
+Notice that:
+- Multiple instances can run with the same scope;
+- An instance without a scope will clean up other running instances, even if they have a defined scope;
+
+To define an instance monitoring scope, use the `--scope-uid` argument or the `WATCHTOWER_SCOPE_UID` environment variable on startup and set the _com.centurylinklabs.watchtower.scope-uid_ label with the same value for the containers you want to include in this instance's scope.
+
+For example, in a Docker Compose config file:
+
+```json
+version: '3'
```

Comment:
Is this a new file @victorcmoura ?
If yes, you also need to add it to the docs menu, by editing `mkdocs.yml`

Reply:
Yes it is! I'll add it there